### ISSUE TITLE
Unclear how to get test methodology information

### ISSUE THREAD

**Issue by bassosimone** [TheTorProject/ooniprobe-android#71]
User story: I am an advanced user, I end up in the Run Tests section where I am clearly hinted to hit the Run button to start tests, then I am hinted to go into the Past Tests section, where I can see results; at this point, I see anomalies that I think should not be anomalies, and I don't have a quick way to see the test methodology from the results (not to mention that it's not obvious to me that I can get info on a specific test by going on the Run Tests section and running it).

(*IMHO* this history seems to suggest that the place where one may want to know about the methodology is indeed when he/she is looking at results, no?)

xref: https://github.com/TheTorProject/ooniprobe-ios/issues/101

**Comment by lorenzoPrimi** [comment]
@bassosimone I guess this can be closed as it's refered to 1.x app

**Comment by bassosimone** [comment]
It's not clear to me whether the new app makes this user story possible!

**Comment by lorenzoPrimi** [comment]
I don't really understand what you try to ask here.
When you finish run a test you go in the test result page, the new test is highlighted.
Then you see some basic information about the test like number of site tested/number blocked, or upload/download speed etc.

**Comment by bassosimone** [comment]
Sorry for being brief. Then, the user request was to have a shortcut to the measurement methodology from the results, so to better evaluate whether it was false positive or not.

**Comment by lorenzoPrimi** [comment]
It's not in the mockup at the moment.
There will be a button to "re-test" an url in case of web censorship test.
I don't think it's needed any button for the other types of test, as they are only two taps away:
- Tap Dashboard
- Tap Run

**Comment by bassosimone** [comment]
Okay, then let's close this issue.

**Comment by hellais** [comment]
Actually I think the new UX does have enough space to fit links to the methodology (on the test details screen) and we should be sure to take this into account when @agrabeli and @elioqoshi work on the copy.

I would say we keep this open and have it as a feature for the new app.

**Comment by lorenzoPrimi** [comment]
@hellais I don't like having an "useless" button for an action that can be achieved only with two taps instead of one.
Discuss.

**Comment by hellais** [comment]
@lorenzoPrimi perhaps I didn't explain myself well. What I am suggesting is that we add to the following screen this button:

![00testoverview websites methodology](https://user-images.githubusercontent.com/424620/37908999-04b8ea38-310a-11e8-8292-78205645e82d.png)

I believe we don't have a link to the methodology anywhere else in the app.

**Comment by lorenzoPrimi** [comment]
@bassosimone is talking about adding a Run button inside test result of a specific test. It's a different thing

**Comment by hellais** [comment]
@bassosimone well that was his suggestion, but what I am saying is that to satisfy the user story of a user obtaining methodology information we should be sure to remember to add a link to the methodology in the test overview (which we currently don't have).

**Comment by elioqoshi** [comment]
Linking the Methodoloy in a Disclaimer style on that screen is fine for me. Let's keep this open until it's implemented.

**Comment by lorenzoPrimi** [comment]
Ok so it's not about the first request of sbs.
Actually we already have a link to the methodoloy in the current app...

**Comment by hellais** [comment]
@elioqoshi it's probably also worth considering if there is some place inside of the test results pages to add another link that points to the test description (the screen I linked above).

**Comment by lorenzoPrimi** [comment]
In current app:
![screenshot_20180326-153729](https://user-images.githubusercontent.com/1616097/37909756-0b920a22-310c-11e8-987b-64bf21f191d0.png)

**Comment by elioqoshi** [comment]
@hellais you mean in the Test Results Summary pages of specific measurements to link to their respective test description?

**Comment by hellais** [comment]
> you mean in the Test Results Summary pages of specific measurements to link to their respective test description?

Correct.

**Comment by bassosimone** [comment]
> but what I am saying is that to satisfy the user story of a user obtaining methodology information we should be sure to remember to add a link to the methodology in the test overview

Actually that was I was suggesting. Too bad I was not able to explain myself well :-/

**Comment by lorenzoPrimi** [comment]
@elioqoshi @agrabeli will be a link to methodology in the description?

**Comment by hellais** [comment]
> will be a link to methodology in the description?

Yes, let's not forget about adding links to the mockups and copy.
